### PR TITLE
[NAE-1703] Dashboard

### PR DESCRIPTION
**Pull request by mazarijuraj**
# Description

Configurable dashboard and dashboard tile processes

Implements [NAE-1703]

## How Has Been This Tested?

Manually

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes have been checked, personally or remotely, with @...
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have resolved all conflicts with the target branch of the PR
- [ ] I have updated and synced my code with the target branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes:
    - [ ] Lint test
    - [ ] Unit tests
    - [ ] Integration tests
- [ ] I have checked my contribution with code analysis tools:
    - [ ] [SonarCloud](https://sonarcloud.io/project/overview?id=netgrif_application-engine)
    - [ ] [Snyk](https://app.snyk.io/org/netgrif/project/ea82b3b8-658d-41f5-89a7-76cd600da01f)
- [ ] I have made corresponding changes to the documentation:
    - [ ] Developer documentation
    - [ ] User Guides
    - [ ] Migration Guides

[NAE-1703]: https://netgrif.atlassian.net/browse/NAE-1703?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/main/resources/petriNets/dashboard.xml [hunk 1]
```diff
@@ -0,0 +1,229 @@
+<document xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://petriflow.com/petriflow.schema.xsd">
+	<id>dashboard</id>
+	<initials>DSH</initials>
+	<title>Dashboard</title>
+	<icon>dashboard</icon>
+	<defaultRole>true</defaultRole>
+	<anonymousRole>true</anonymousRole>
+	<transitionRole>false</transitionRole>
+	<data type="text" immediate="true">
```
Comment (minop) [inline src/main/resources/petriNets/dashboard.xml:9]: data variables lack i18n

File: src/main/resources/petriNets/dashboard.xml [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+		<title/>
+		<component>
+			<name>dashboard</name>
+		</component>
+	</data>
+	<transition>
+		<id>t1</id>
+		<x>380</x>
+		<y>180</y>
+		<label>Configure</label>
```
Comment (minop) [inline src/main/resources/petriNets/dashboard.xml:38]: transitions lack i18n

File: src/main/resources/petriNets/dashboard_tile.xml [hunk 3]
```diff
@@ -0,0 +1,1263 @@
+<document xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://petriflow.com/petriflow.schema.xsd">
+    <id>dashboard_tile</id>
+    <initials>DST</initials>
+    <title name="model_title">Dashboard Tile</title>
```
Comment (minop) [inline src/main/resources/petriNets/dashboard_tile.xml:4]: i18n keys are defined in the whole process but translations are missing

**Review by minop — DISMISSED** [review]

File: src/main/resources/petriNets/dashboard.xml (hunk 1 above)
Comment (mazarijuraj) [reply src/main/resources/petriNets/dashboard.xml:10]: done

File: src/main/resources/petriNets/dashboard.xml (hunk 2 above)
Comment (mazarijuraj) [reply src/main/resources/petriNets/dashboard.xml]: done

File: src/main/resources/petriNets/dashboard_tile.xml (hunk 3 above)
Comment (mazarijuraj) [reply src/main/resources/petriNets/dashboard_tile.xml:4]: done

**Pull request merged**